src/huggingface_hub/utils/_telemetry.py

```diff
@@ -100,7 +100,12 @@ def _send_telemetry_in_thread(
     library_version: Optional[str] = None,
     user_agent: Union[Dict, str, None] = None,
 ) -> None:
-    """Contains the actual data sending data to the Hub."""
+    """Contains the actual data sending data to the Hub.
+    
+    This function is called directly in gradio's analytics because
```

Comment:
Is it gradio's analytics or only gradio lite? Can you also add a link to the 
Gradio issue as a comment? Thanks a lot!

Reply:
It's gradio's !